graphstorm-processing/tests/test_dist_numerical_transformation.py

```diff
@@ -297,3 +298,67 @@ def test_multi_numerical_transformation_with_array_input(spark: SparkSession, ch
     transformed_rows = transformed_df.collect()
     for row, expected_vector in zip(transformed_rows, expected_vals):
         assert_array_almost_equal(row["feat"], expected_vector, decimal=3)
+
+
+def rank_gauss(feat, eps):
+    lower = -1 + eps
+    upper = 1 - eps
+    range = upper - lower
+    i = np.argsort(feat, axis=0)
+    j = np.argsort(i, axis=0)
+    j_range = len(j) - 1
+    divider = j_range / range
+    feat = j / divider
+    feat = feat - upper
+    return erfinv(feat)
+
+
+@pytest.mark.parametrize("epsilon", [0.0, 1e-6])
+def test_rank_gauss(spark: SparkSession, check_df_schema, epsilon):
+    data = [("mark", 0.0, None), ("john", 15.0, 10000), ("tara", 26.0, 20000), ("jen", 40.0, 10000)]
+
+    columns = ["name", "age", "salary"]
```

Comment:
We don't need the extra columns here right? Can just have data have the float values, and have a single "value" column.